connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -230,14 +230,6 @@ class PlanGenerationTestSuite
   private def temporals = createLocalRelation(temporalsSchemaString)
 
   /* Spark Session API */
-  test("sql") {
```

Comment:
Pls also delete the corresponding auto generated files too. Each test should have 3 files related to it. `.json`, `.bin`, `.explain`

Reply:
Done.